[IMP] Module to install all RMA module (APP)

yaniaular opened the pull request:
Module to install all RMA module (APP)

yaniaular commented:
We are working on this PR to make it perfect to merge with OCA/rma.
The modules presented here are part of a collection of modules for the operations of RMA improved by Vauxoo. Each module will be presented in separate PRs because the PR #26 was too heavy for review.. Some dependencies needed.

pedrobaeza commented on rma/README.rst:
```diff
@@ -0,0 +1,13 @@
+App RMA
```
Use full OCA's template

pedrobaeza commented on rma/i18n/es.po:
```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```
I think you don't need these files as this module doesn't have any translation

nhomar replied:
2015-08-05 15:48 GMT-05:00 Pedro M. Baeza notifications@github.com:

> I think you don't need these files as this module doesn't have any
> translation

The same than before, we use them in production with a "NO WARNING RULE"
and this company use 3 spanish from time to time they will have little
diferences but for now the empty file save us the warning of not transated
elements.

It save us a little problems of have false warnings, just warnings that
muct be taken into account.

Do you think you can accept such situation?

## 

---

Saludos Cordiales

CEO at Vauxoo https://www.vauxoo.com Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
http://about.me/nhomar?promo=email_sig

pedrobaeza replied:
Odoo complains in this case? I'm just asking, because I don't know

nhomar replied:
2015-08-05 16:34 GMT-05:00 Pedro M. Baeza notifications@github.com:

> Odoo complains in this case? I'm just asking, because I don't know

Yes it returns a WARNING in the log when you update a module without
translations in a lang not installed.

## 

---

Saludos Cordiales

CEO at Vauxoo https://www.vauxoo.com Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
http://about.me/nhomar?promo=email_sig

pedrobaeza replied:
Ah, OK, thanks for the info.

pedrobaeza commented on rma/__openerp__.py:
```diff
@@ -0,0 +1,54 @@
+    "installable": True,
+    "auto_install": False
+}
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Remove this line

gurneyalex commented:
CI is red -> needs fixing.

Commit:
[IMP] Module to install all RMA module (APP)

[REF] rma: change dependency crm_rma_location_rma -> crm_rma_location

[REF] rma: remove crm_rma_advance_location from dependencies

nhomar commented:
@yaniaular can you update the status I think we finished all necesary PR for this to be merged, but not sure.

yaniaular commented:
@nhomar when runbot will be green, it is ready to merge

yaniaular commented:
@nhomar @gurneyalex @pedrobaeza Please review and merge

nhomar commented:
+1

yaniaular commented:
@gurneyalex @pedrobaeza Please review and merge

gurneyalex commented:
:+1:

RoelAdriaans commented:
:+1:
